js/js.tests/test/org/jetbrains/kotlin/js/test/BasicWasmBoxTest.kt

```diff
@@ -48,8 +51,20 @@ abstract class BasicWasmBoxTest(
 
     private val spiderMonkey by lazy { SpiderMonkeyEngine() }
```

Comment:
Looks like SpiderMonkey is no longer used

Reply:
Yes. Kept it in case we would want to use them later when they implement recent spec.

Reply:
got it